README.md

```diff
@@ -21,84 +21,73 @@ This project was born from the following needs:
   even if your infrastructure management tool doesn't provide that
   feature
 
-## DOCUMENTATION
+## What is a brick
+First, exeIaC deals with infra bricks. What is an infra brick ? Basically
+it's a directory that contains some infra code. It can be a terraform directory
+to deploy a VM, an ansible playbook to configure an host, an helm chart or simply
+a template that describe some instruction to do manually.
 
-- philosophy: theoretical approach useful to write your infra code and understand
-  exeiac best practices
-  - common infra code problems we try to solve with exeiac
-  - define infra as a set of bricks
-  - define the brick concept
-  - explain how bricks depends of each other
-  - vocabulary
-- development: contain specs and schema to understand how it is coded
-- user: contain all you need to use the tool and create an infra code that
-  respect the convention and best practices
-- examples: examples of simple infra code and module
+To understand this page it's sufficient. But if you want to understand deeper 
+the genious idea behind that concept, how many type of dependencies exist 
+between your bricks and all element that compose your brick (code description,
+input, ouput, events, trigger, command), I let you read that [page](./docs/brick_concept_and_dependencies)
 
-## Get started
+## Get started and how it works summary
```

Comment:
```suggestion
## Get started
```

I think this is still just a get started paragraph.

The ideal case here would be to have a `## tutorial` section or file with an actual step by step with a toy infrastructure and the content of the files so a user can get an actual running project to play with. Basically a more involved guide than what already is in `howto_write_configuration_file`. (c.f. [a tutorial's purpose](https://documentation.divio.com/tutorials.html) and the [4 types of doc](https://www.writethedocs.org/videos/eu/2017/the-four-kinds-of-documentation-and-why-you-need-to-understand-what-they-are-daniele-procida/))

If you provide me with a simple running project, I can gladly draft something for you :slightly_smiling_face: